crates/indexer/src/util.rs

```diff
@@ -6,6 +6,8 @@ use bigdecimal::{FromPrimitive, Signed, ToPrimitive, Zero};
 use serde_json::Value;
 use sha2::Digest;
 
+pub const MAX_TIMESTAMP_SECS: i64 = 253_402_300_799; // 9999-12-31 23:59:59;
```

Comment:
is this based on the iso standard? if so pls add it in comment as well, if not, lets base it on that :-P

Reply:
oh yea my new comment didn't commit. updated.